Jenkinsfile

```
@@ -3,9 +3,9 @@
 @Library('OGJenkinsLib@3.7.0') _
 
 final GIT_REPOSITORY_NAME = 'amazon-eks-ami'
-final KUBERNETES_VERSION = '1.13.7'
+final KUBERNETES_VERSION = '1.15.10'
```

```suggestion
final KUBERNETES_VERSION = '1.15.11'
```
is the latest version available https://docs.aws.amazon.com/eks/latest/userguide/kubernetes-versions.html

---

bumped to 11

---

still says `.10`